tests/rptest/services/provider_clients/rpcloud_client.py

```diff
@@ -91,12 +91,18 @@ def _http_get(self,
         else:
             return _r.json()
 
-    def _http_post(self, base_url=None, endpoint='', **kwargs):
+    def _http_post(self,
+                   base_url=None,
+                   endpoint='',
+                   override_headers=None,
+                   **kwargs):
         token = self._get_token()
         headers = {
             'Authorization': f'Bearer {token}',
             'Accept': 'application/json'
         }
+        if override_headers:
```

Comment:
Style: this is fine but a slightly cleaner pattern is to set `override_headers` to `{}` as a default then just do `headers = { ... } | override_headers`